src/main/java/com/commercetools/project/sync/SyncModuleOption.java

```diff
@@ -0,0 +1,56 @@
+package com.commercetools.project.sync;
+
+import com.commercetools.project.sync.util.SyncUtils;
+import com.commercetools.sync.cartdiscounts.CartDiscountSync;
+import com.commercetools.sync.categories.CategorySync;
+import com.commercetools.sync.commons.BaseSync;
+import com.commercetools.sync.customobjects.CustomObjectSync;
+import com.commercetools.sync.inventories.InventorySync;
+import com.commercetools.sync.products.ProductSync;
+import com.commercetools.sync.producttypes.ProductTypeSync;
+import com.commercetools.sync.states.StateSync;
+import com.commercetools.sync.taxcategories.TaxCategorySync;
+import com.commercetools.sync.types.TypeSync;
+import java.util.Arrays;
+import java.util.stream.Stream;
+
+public enum SyncModuleOption {
```

Comment:
Is it better to use plural form? since more than one option provided.

Reply:
I don't think so. In Java manual, it's also in singular form: https://docs.oracle.com/javase/tutorial/java/javaOO/enum.html

Reply:
Yes. I checked. that's why i set it outdated